index.html

```diff
@@ -219,5 +219,27 @@ <h2>Skills</h2>
         </ul>
       </div>
       </section> 
+
+      <section class="contact-section">
+        <div class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+            <input class="form-input" type="text" name="name" placeholder="Full name">
+            <input class="form-input" type="email" name="email" placeholder="Email address">
+            <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here"></textarea>
+            <input class="send" type="submit" value="Get in touch">
```

Comment:
- [ ] Additionally, let's focus on making the form inputs more professional and easily accessible. Again we shall refer to the project requirements. I request to put the attached points into consideration.
- Make sure that:
  - Name field should accept up to 30 chars.
  - Message field accepts up to 500 chars.
  - A user should not be able to submit the form if one or more of the fields is empty.